frontend/src/api/window.js

```diff
@@ -225,3 +225,48 @@ export function getPrintingOptions({ entity, windowId, docId, tabId, rowId }) {
       '/printingOptions'
   );
 }
+
+/**
+ * @method initQuickActions
+ * @summary Fetch data for table quick input
+ * @param {string} entity - for example 'window'
+ * @param {string} windowId
+ * @param {string} docId
+ * @param {string} tabId
+ * @param {string} subentity
+ */
+export function initQuickActions(entity, windowId, docId, tabId, subentity) {
+  return post(
+    `${config.API_URL}/${entity}/${windowId}/${docId}${
+      tabId ? `/${tabId}` : ''
+    }${subentity ? `/${subentity}` : ''}`
+  );
+}
+
+/**
+ * @method completeRequest
+ * @summary Save changes in attributes/quick input
+ * @param {string} entity - for example 'window'
+ * @param {string} windowId
+ * @param {string} docId
+ * @param {string} tabId
+ * @param {string} subentity
+ * @param {string} subentityId
+ */
+export function completeRequest(
+  entity,
+  docType,
+  docId,
+  tabId,
+  rowId,
+  subentity,
+  subentityId
+) {
+  return post(
+    `${config.API_URL}/${entity}${docType ? `/${docType}` : ''}${
```

Comment:
Maybe we could make a bit of cleaning in here and have the values to be sent put in constants before forming the post.

`const docTypeParam = docType ? docType : '';`
...
and have in the end a cleaner thing

return post(`${config.API_URL}/${docTypeParam}/${docIdParam}/${tabIdParam}/${subEntity}/${entityId});